app/assets/stylesheets/components/_signup-link.scss

```diff
@@ -0,0 +1,34 @@
+.app-c-signup-link--with-background-and-border {
+  padding: govuk-spacing(6);
+  background-color: govuk-colour("light-grey", $legacy: "grey-4");
+  border: 1px solid $govuk-border-colour;
+}
+
+.app-c-signup-link__link {
+  @include govuk-font(19);
+}
+
+.app-c-signup-link--link-only .app-c-signup-link__link {
```

Comment:
I'm definitely nitpicking now... but could you reorder these classes a little? I'd put all the `__elements` first and the `--modifiers` after that.